doc/api/assert.markdown

```diff
@@ -85,9 +85,27 @@ Custom error validation:
       "unexpected error"
     );
 
-## assert.doesNotThrow(block[, message])
+## assert.doesNotThrow(block[, error][, message])
 
-Expects `block` not to throw an error, see `assert.throws` for details.
+Expects `block` not to throw an error. See `assert.throws` for more details.
```

Comment:
Two more things (sorry!):

Should probably be `assert.throws()` with the parentheses, and that text should probably link to the section of the document for `assert.throws()`.